airflow/providers/google/cloud/triggers/dataplex.py

```diff
@@ -0,0 +1,110 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module contains Google Dataplex triggers."""
+from __future__ import annotations
+
+import asyncio
+from typing import AsyncIterator, Sequence
+
+from google.cloud.dataplex_v1.types import DataScanJob
+
+from airflow.providers.google.cloud.hooks.dataplex import DataplexAsyncHook
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+
+class DataplexDataQualityJobTrigger(BaseTrigger):
+    """
+    DataplexDataQualityJobTrigger runs on the trigger worker and waits for the job to be `SUCCEEDED` state.
+
+    :param job_id: Optional. The ID of a Dataplex job.
+    :param project_id: Google Cloud Project where the job is running
+    :param data_scan_id: Required. DataScan identifier.
```

Comment:
```suggestion
    :param data_scan_id: Required. DataScan identifier.
    :param project_id: Google Cloud Project where the job is running.
```
I prefer to keep params in docstring and constructor in the same order

Reply:
done